iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergingView.java

```diff
@@ -289,12 +277,11 @@ public int getCountryCount() {
 
     @Override
     public Identifiable<?> getIdentifiable(final String id) {
-        return index.getNetworkStream()
-                .map(n -> n.getIdentifiable(id))
-                .filter(Objects::nonNull)
-                .map(index::getIdentifiable)
-                .findFirst()
-                .orElse(null);
+        Identifiable<?> search = index.getMergedLine(id);
+        if (Objects.isNull(search)) {
+            search = index.get(n -> n.getIdentifiable(id), index::getIdentifiable);
+        }
+        return search;
```

Comment:
I think you can use an Optional there. Something like this:
```
return Optional.ofNullable(index.getMergedLine()).orElse(index.get(...));
```

Tell me if it's cleaner (more readable) or keep like this and ignore the next comments otherwise